x-pack/legacy/plugins/spaces/server/lib/copy_to_spaces/copy_to_spaces.test.ts

```diff
@@ -0,0 +1,393 @@
+                "highWaterMark": 16,
+                "length": 0,
+                "needReadable": false,
+                "objectMode": true,
+                "paused": false,
+                "pipes": null,
+                "pipesCount": 0,
+                "readableListening": false,
+                "reading": false,
+                "readingMore": false,
+                "resumeScheduled": false,
+                "sync": false,
+              },
+              "readable": false,
+            },
+            "savedObjectsClient": null,
+            "supportedTypes": Array [
+              "dashboard",
+              "visualization",
+            ],
+          },
+        ],
+        Array [
+          Object {
+            "namespace": "destination2",
+            "objectLimit": 1000,
+            "overwrite": true,
+            "readStream": Readable {
+              "_events": Object {
+                "data": [Function],
+                "end": [Function],
+                "error": [Function],
+              },
+              "_eventsCount": 3,
+              "_maxListeners": undefined,
+              "_read": [Function],
+              "_readableState": ReadableState {
+                "awaitDrain": 0,
+                "buffer": BufferList {
+                  "head": null,
+                  "length": 0,
+                  "tail": null,
+                },
+                "decoder": null,
+                "defaultEncoding": "utf8",
+                "destroyed": false,
+                "emitClose": true,
+                "emittedReadable": false,
+                "encoding": null,
+                "endEmitted": true,
+                "ended": true,
+                "flowing": true,
+                "highWaterMark": 16,
+                "length": 0,
+                "needReadable": false,
+                "objectMode": true,
+                "paused": false,
+                "pipes": null,
+                "pipesCount": 0,
+                "readableListening": false,
+                "reading": false,
+                "readingMore": false,
+                "resumeScheduled": false,
+                "sync": false,
+              },
+              "readable": false,
+            },
+            "savedObjectsClient": null,
+            "supportedTypes": Array [
+              "dashboard",
+              "visualization",
+            ],
+          },
+        ],
+      ]
+    `);
+  });
+
+  it(`doesn't stop copy if some spaces fail`, async () => {
+    const { savedObjectsClient, savedObjectsService } = setup({
+      objects: [
+        {
+          type: 'dashboard',
+          id: 'my-dashboard',
+          attributes: {},
+        },
+        {
+          type: 'visualization',
+          id: 'my-viz',
+          attributes: {},
+        },
+        {
+          type: 'index-pattern',
+          id: 'my-index-pattern',
+          attributes: {},
+        },
+      ],
+      importSavedObjectsImpl: opts => {
```

Comment:
It does... my thinking here was that it wasn't creating any meaningful gaps, since the override is only used to test specific failure conditions. I'll update regardless.